src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -60,17 +60,13 @@
               </a>
             </mat-list-item>
             <mat-list-item *ngIf="!orgObj || orgObj?.published.length === 0">
-              <p data-cy="no-published-apptool-message" *ngIf="pageEntryType === EntryType.AppTool" class="text-center size-small">
-                No published tools.
-              </p>
-              <p
-                [attr.data-cy]="'no-published-' + pageEntryType + '-message'"
-                *ngIf="
+              <p [attr.data-cy]="'no-published-' + pageEntryType + '-message'" class="text-center size-small">
+                No published
+                {{
                   pageEntryType === EntryType.Service || pageEntryType === EntryType.BioWorkflow || pageEntryType === EntryType.Notebook
```

Comment:
This could be simplified to `pageEntryType == EntryType.AppTool ? 'tool' : pageEntryType`